cmd/util.go

```diff
@@ -76,3 +79,21 @@ func getDefaultAuthTokenPath() (string, error) {
 
 	return token, nil
 }
+
+func getAuthToken(cmd *cobra.Command) (string, error) {
+	tokenValue, err := cmd.Flags().GetString(argRootTokenValue)
+	if err != nil {
+		return "", err
+	} else if tokenValue != "" {
+		return tokenValue, nil
+	}
+	tokenPath, err := getDefaultAuthTokenPath()
+	if err != nil {
+		return "", err
+	}
+	token, err := ioutil.ReadFile(tokenPath)
+	if err != nil {
+		return "", errors.Wrap(err, "Please Login first")
```

Comment:
I suggest to change `Please Login first` to something more descriptive: `Failed to read token from file`

Reply:
I don't think it is a good idea: if the file is not there, it means you need to log in.
This is how it was before my PR, I simply moved the code.